### PR TITLE
Use `v*.*.*` branches to trigger github actions instead of `development`

### PR DESCRIPTION
**Pull request by KPrasch**

### CODE REVIEW

File: .github/workflows/darker.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
     branches:
       - main
-      - development
+      - v*
```
Comment (derekpierre) [inline .github/workflows/darker.yml:7]: Did you want to do `v*.*.*`? Otherwise any branch starting with `v` will match...

Comment (KPrasch) [reply .github/workflows/darker.yml:7]: tbh I like the simplicity and we can likely avoid other types of upstream branches that start with "v".

Comment (KPrasch) [reply .github/workflows/darker.yml]: updated to use `v*.*.*` ✔️

**Comment by KPrasch** [comment]
> Does the branch therefore symbolize the major/minor version, but tags would now only be used for updates to the patch number...?

We can still tag both minor and patch releases as they get merged.  Patch (bugfix) commits can use a PR based directly over the version they are patching.

**Review by derekpierre — APPROVED** [review]
🎸

**Pull request merged**